.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+    mocha: true
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  plugins: ['@typescript-eslint', 'prettierx'],
+  extends: [
+    'standard',
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettierx/standardx',
+    'plugin:prettierx/@typescript-eslint'
+  ],
+  rules: {
+    'no-void': 'off',
+
+    // Disabled because we intentionally have any for generics
+    '@typescript-eslint/no-explicit-any': 'off',
+
+    // Disabled because it reports these for js files
+    '@typescript-eslint/no-var-requires': 'off',
```

Comment:
I think that one could be easily removed by fixing the JS files.

Reply:
Yes, but I would prefer (that someone) do this in a separat PR